src/parameterizations/vertical/MOM_geothermal.F90

```diff
@@ -119,6 +126,7 @@ subroutine geothermal(h, tv, dt, ea, eb, G, GV, CS, halo)
   Angstrom  = GV%Angstrom_H
   H_neglect = GV%H_subroundoff
   p_ref(:)  = tv%P_Ref
+  Idt       = 1/dt
```

Comment:
Should use `1.0` or `1.` to indicate a floating point operation.

Also spaces between operators is good practice, e.g. `1.0 / dt`